app/src/test/java/com/kickstarter/viewmodels/PledgeFragmentViewModelTest.kt

```diff
@@ -561,7 +561,7 @@ class PledgeFragmentViewModelTest : KSRobolectricTestCase() {
         this.shippingSummaryIsGone.assertNoValues()
         this.totalDividerIsGone.assertValue(false)
 
-        this.lakeTest.assertValue(EventName.PAGE_VIEWED.eventName)
+        this.lakeTest.assertValues(EventName.PAGE_VIEWED.eventName, EventName.PAGE_VIEWED.eventName)
```

Comment:
With this assert, you are checking that the event is launch two times. That is happening because the code is duplicated. Remove the duplicated part, and let the asserts checking that the event has been triggered just once again.

Reply:
done